faq.md

```diff
@@ -10,9 +10,7 @@ permalink: /faq/
 
 ## Can I get an exe?
 
-**NO**, not yet. Xenia is in its extremely early phases and not ready for use by
-non-developers. If you can't build it yourself using the instructions provided
-it probably means you should wait until alpha/beta/etc.
+Yes, by going to the latest build <a href="https://ci.appveyor.com/project/benvanik/xenia/build/artifacts">here</a> for Windows. From there go to Jobs > Configuration Release > Artifacts than you can dowload the zip file.  
```

Comment:
"[...] Release > Artifacts than you can [...]" Change "than" to "then"

Reply:
@DrChat  Is this good to go?